pkgs/applications/science/electronics/kingstvis/default.nix

```
@@ -0,0 +1,61 @@
+{ stdenv
+, buildFHSEnv
+, lib
+, fetchzip
+, glib
+, xorg
+, libGL
+, zlib
+, fontconfig
+, freetype
+, xkeyboard_config
+, dbus
+}:
+
+let
+
+  name = "kingstvis";
+  version = "3.6.1";
+  src = fetchzip {
+    url = "http://res.kingst.site/kfs/KingstVIS_v${version}.tar.gz";
+    hash = "sha256-eZJ3RZWdmNx/El3Hh5kUf44pIwdvwOEkRysYBgUkS18=";
+  };
+
+in
+
+buildFHSEnv {
+  inherit name;
```

```suggestion
  inherit pname version;
```

---

buildFHSEnv requires the "name" property.
Won't work with "pname" and "version" combo.

---

```suggestion
  name = "${pname}-${version}"
  inherit pname version;
```
we can solve this by doing something like this

---

Can't add both "name" and pname+version
```sh
error: assertion '(((pname != null) || (version != null)) -> ((name == null) && (pname != null)))' failed

       at /nix/store/ikxgkyxy4zqzlazj03jshl5rjfd7kxcx-source/pkgs/build-support/build-fhsenv-bubblewrap/default.nix:30:1:

           29|
           30| assert (pname != null || version != null) -> (name == null && pname != null); # You must declare either a name or pname + version (preferred).
             | ^
           31|
```

Doing 
```nix
name = "${pname}-${version}";
```
will produce an executable named "kingstvis-3.6.1".
Is that ok?

---

whatever, we just leave it out